community/cloud-foundation/templates/cloudbuild/examples/cloudbuild_reposource.yaml

```diff
@@ -0,0 +1,42 @@
+# Example of the Cloud Build template usage.
+#
+# In this example, a Cloud Build with the source in a Cloud Source 
+# repository is triggered.
+#
+# Replace the following placeholders with valid values:
+#   <FIXME:project_id>  : PROJECT_ID of the project where the repo resides
+#   <FIXME:repo_name>   : a valid Cloud Source repository name
+#   <FIXME:branch_name> : a valid branch name in the source repository
+#   <FIXME:builder_name>: a cloud-builder supported on Google Cloud;
+#     see details at https://cloud.google.com/cloud-build/docs/cloud-builders
+#   <FIXME:build_args>  : a list of build arguments supported by the builder;
+#     enter each argument on a separate line as show below
+#     Example git clone build step:
+#     - name: 'gcr.io/cloud-builders/git'
+#       args:
+#         - clone
+#         - https://github.com/GoogleCloudPlatform/deploymentmanager-samples
+
+imports:
+  - path: templates/cloudbuild/cloudbuild.py
+    name: build.py
+
+resources:
+  - name: mycloudbuild
+    type: build.py
+    properties:
+      source:
+        repoSource:
+          projectId: <FIXME:project_id>
+          repoName: <FIXME:repo_name>
+          branchName: <FIXME:branch_name>
+      steps:
+        - name: 'gcr.io/cloud-builders/<FIXME:builder_name>'
```

Comment:
aren't we creating the build steps here? we should pass in the builder name
also I don't believe we need this amount of `FIXME`s; pre-populate as much as possible with build and push steps. Likely the only `FIXME` you need is is the `dir` and `repoName` for anything that is not targeting something the user should must create prior to running the example we should be able to put a value. Also please call out on the example description the pre-work required (i.e. have a source repo etc )

Reply:
Cleaned this up good. Now the only `FIXME` is the `repo_name`
Fix in commit `b806511`